src/main/java/iudx/resource/server/apiserver/AsyncRestApi.java

```diff
@@ -132,6 +132,7 @@ private void handleAsyncSearchRequest(RoutingContext routingContext) {
             LOGGER.debug("Info: IUDX json query;" + json);
             JsonObject requestBody = new JsonObject();
             requestBody.put("ids", json.getJsonArray("id"));
+            json.put("someKey", Math.random());
```

Comment:
purpose of this random number in JSON?

Reply:
it would create a different requestId every time so that process4NewRequestId() method is called and we see the csv output in a file during the execution of asynScroll(). This line is only used for demonstration of the flow... This would be removed in the future.

Reply:
Will remove Math.random() to upload it to S3 bucket